tests/ops/integration_tests/setup_scripts/postgres_setup.py

```diff
@@ -15,6 +15,7 @@
 
 # Need to manually import this model because it's used in src/fides/api/models/property.py
 # but that file only imports it conditionally if TYPE_CHECKING is true
+from fides.api.models.experience_notices import ExperienceNotices
```

Comment:
Seemed to need this now for bringing up a postgres database locally? We need to figure out our imports.